Documentation/writing-tizen-csharp-apps.md

```diff
@@ -0,0 +1,131 @@
+# Writing a Tizen .NET Application
+
+This document provides you how to write a Tizen .NET Application with Machine Learning APIs.
+Since Tizen 5.5 (not released yet), Machine Learning Inference functionality has been provided on Mobile, Wearable and TV profile.
+
+## Installing Visual Studio Tools for Tizen
+In order to use this functionality, you need to install Visual Studio Tools for Tizen and Tizen SDK version 5.5 or higher. You can find the detailed guide for this at the below link.
+
+* https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen
+
+
+# Machine Learning Inference
+
+The Machine Learning Inference features introduce how you can easily invoke the neural network model and get the inference output result.
+
+With Tizen.MachineLearning.Inference.SingleShot, you can load the existing neural network model or your own specific model from storage and invoke it with a single instance of input data, finally get the inference output result.
+
+Currently, this is an experimental functionality and is likely to be changed.
+
+The main features of the Tizen.MachineLearning.Inference are:
+
+- Managing a Tensor information
+
+  You can [configure the input and output Tensor Information](#manage) such as its name, data type and dimension.
+
+- Loading the neural network model and configures a runtime environment
+
+  You can [load the neural network model from storage and configure a runtime environment](#load).
+
+- Invoking the neural network model with the input data
+
+  After setting up the SingleShot instance with its required information, you can [invoke the model with the input data and get the inference output result](#invoke).
+
+- Fetching the inference result after invoking
+
+  You can [fetch the inference result after invoking](#fetch).
+
+## Prerequisites
+
+To enable your application to use the Machine Learning Inference functionality:
```

Comment:
To use Machine Learning Inference with your application:

Reply:
ACK!